lib/scripts/services/contextMenu.js

```diff
@@ -1,19 +1,57 @@
+function _checkIfWordCountNotOne(inputString){
```

Comment:
@rohankatyal29 why are these two functions outside the class scope. Right now they aren't class functions. You can put these inside the class and access them as `this._currentURLIsBlacklistedOrMTWIsOff`